istioctl/pkg/verifier/verifier.go

```diff
@@ -150,21 +151,30 @@ func (v *StatusVerifier) verifyInstallIOPRevision() error {
 		}
 		return fmt.Errorf("could not load IstioOperator from cluster: %v. Use --filename", err)
 	}
-	if v.manifestsPath != "" {
-		iop.Spec.InstallPackagePath = v.manifestsPath
-	}
-	profile := manifest.GetProfile(iop)
-	by, err := yaml.Marshal(iop)
-	if err != nil {
-		return err
-	}
-	mergedIOP, err := manifest.GetMergedIOP(string(by), profile, v.manifestsPath, v.controlPlaneOpts.Revision, v.client, v.logger)
-	if err != nil {
-		return err
+	var crdTotal, istioDeploymentTotal int
+	multiErr := &multierror.Error{}
+	for _, iop := range iops {
```

Comment:
We've already had one integration test for `verify-install`, and this bug-fix is just combining all iop results together instead of randomly choosing one.